main.py

```diff
@@ -2,6 +2,48 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    actors = [
+        ("George", "Clooney"),
+        ("Keanu", "Reeves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+    genres = ["Western", "Action", "Drama"]
+
+    for item_first, item_second in actors:
+        Actor.objects.create(
```

Comment:
You're very clever, you wrote down the first and last names separately)). But please label the variables unpacked from the tuple according to their contents